funcx_endpoint/tests/integration/test_config.py

```diff
@@ -19,8 +19,10 @@
     print("Loading : ", config)
     # Set script dir
     config.provider.script_dir = working_dir
-    config.provider.channel.script_dir = os.path.join(working_dir, "submit_scripts")
-    config.provider.channel.makedirs(config.provider.channel.script_dir, exist_ok=True)
+    config.provider._channel.script_dir = os.path.join(working_dir, "submit_scripts")
+    config.provider._channel.makedirs(
+        config.provider._channel.script_dir, exist_ok=True
+    )
```

Comment:
Is this a desired change? I'm confused about how it's related.

Reply:
Tests sort of needed this at some point, I can take a look again. Might not be super important.